.travis.yml

```diff
@@ -3,4 +3,4 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/ma
 script: bash -ex .travis-opam.sh
 sudo: true
 env:
-        - OCAML_VERSION=4.02 PACKAGE=xapi-idl EXTRA_REMOTES=git://github.com/xapi-project/opam-repo-dev
+        - OCAML_VERSION=4.02 PACKAGE=xapi-idl EXTRA_REMOTES=git://github.com/xapi-project/xs-opam
```

Comment:
Looking at https://github.com/ocaml/ocaml-ci-scripts/blob/master/.travis-ocaml.sh I think we should use `BASE_REMOTE` in place of `EXTRA_REMOTES`